### PR TITLE
[Media] added search by language functionality

### PR DESCRIPTION
**Pull request by ridz1208**
This builds on the existent language flag in the media uploader to add a serach by language functionality to the browsing tab.

additions: 
- filter dropdown
- table column

### CODE REVIEW

**Comment by ridz1208** [comment]
@PapillonMcGill do you mind testing/reviewing this as you implemented the language for the upload?

**Comment by davidblader** [comment]
@ridz1208 can you add a space at the end [here](https://github.com/aces/Loris/blob/minor/modules/media/ajax/FileUpload.php#L297)

File: modules/media/php/media.class.inc [hunk 1]
```diff
@@ -103,6 +103,13 @@ class Media extends \NDB_Menu_Filter
             $fileTypeList[$filetype['file_type']] = $filetype['file_type'];
         }
 
+        //Language selector
+        $languages    = \Utility::getLanguageList();
+        $languageList = [];
```
Comment (HenriRabalais) [inline modules/media/php/media.class.inc]: shouldn't this be `array();` ?
Same thing with line `221`.

File: modules/media/php/media.class.inc [hunk 2]
```diff
@@ -141,8 +149,10 @@ class Media extends \NDB_Menu_Filter
     {
         $user =& \User::singleton();
         // the base query
-        $query  = " FROM media m LEFT JOIN session s ON m.session_id = s.ID".
-        " LEFT JOIN candidate c ON c.CandID=s.CandID";
+        $query  = " FROM media m 
+                    LEFT JOIN session s ON m.session_id = s.ID 
+                    LEFT JOIN candidate c ON c.CandID=s.CandID 
+                    LEFT JOIN language l USING (language_id) ";
         $query .= " WHERE 1=1 ";
```
Comment (HenriRabalais) [inline modules/media/php/media.class.inc]: Why is this line necessary?

File: modules/media/php/media.class.inc (hunk 1 above)
Comment (driusan) [reply modules/media/php/media.class.inc]: Noooooooooooooooooooooooooooooooooooo!

File: modules/media/php/media.class.inc (hunk 2 above)
Comment (driusan) [reply modules/media/php/media.class.inc]: I haven't looked into this module in particular, but many modules include a "WHERE 1=1" clause at the end so that when further clauses get added with "... AND x=foo", it doesn't result in a syntax error.
